lib/rubocop/cop/packaging/require_relative_to_lib.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module RuboCop # :nodoc:
+  module Cop # :nodoc:
+    module Packaging # :nodoc:
+      # This cop is used to identify the `require_relative` calls,
+      # mapping to the "lib" directory and suggests to use `require`
+      # instead.
```

Comment:
We definitely need to add documentation for cops that not include includes the "what", but also the "why". We can do that as a follow up PR though. We could consider making a `ruby-packaging-guide` with guidelines matching the cops in here, just like core cops are implemented to match practices detailed in the `ruby-style-guide`.

Reply:
Yeah, sure!
I was considering to work on #3 (which you raised, sometime back). It should have those "why" and "what" included.

I want to keep it separate from here because if you see all the cops (rubocop, and its extensions), it just explains what it does and the good and bad examples.

If you concur, we can have it the next documentation PR?

Reply:
Absolutely!